### PR TITLE
Bugfix: validate array item

### LINKED ISSUE #372: ArrayInput: Add element to array when switching focus without pressing ENTER
**Describe the bug**

After entering text inside an ArrayInput, I have to press ENTER to add an element to the Array.

I expected the value to also be added to the Array if I switch focus without pressing ENTER, for example:

* When I focus another input in the ComponentDetailsPanel.
* When I click on an empty space in the draw view.
* When I focus another window in my operating system.
* When I click on the "SAVE" button.

**To Reproduce**

https://github.com/ditrit/leto-modelizer/assets/20184007/8a8554c4-ddb0-4155-ac59-de73538e7043

**Application**

- `leto-modelizer` commit hash: `f59e63e9d861fb5d0f0849dc7983be3a76bd779a`.

### PR DESCRIPTION
**Pull request by Zorin95670**
Fix #372

### CODE REVIEW

**Review by aboubakar14 — APPROVED** [review]

File: changelog.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   * Rework on tag display, use tag language for plugin selection on create diagram from template.
 * Draw page:
   * Rework on left drawer.
+  * Re-design attribute field.
```
Comment (k-y0u) [inline changelog.md:47]: which one ?

File: changelog.md [hunk 2]
```diff
@@ -44,6 +44,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  * Re-design information icon on attribute field.
```
Comment (k-y0u) [inline changelog.md:48]: same

File: changelog.md (hunk 1 above)
Comment (Zorin95670) [reply changelog.md]: done

File: changelog.md (hunk 2 above)
Comment (Zorin95670) [reply changelog.md]: done

**Review by k-y0u — APPROVED** [review]

**Pull request merged**